python-rpmautospec.spec

```diff
@@ -37,26 +36,27 @@ Requires: python3-pygit2
 
 %install
 %py3_install
-#install -m 0644 koji_plugin/rpmautospec_plugin.py %{buildroot}/usr/lib/koji-hub-plugins/
+mkdir -p  %{buildroot}%{_libdir}/koji-hub-plugins/
+install -m 0644 koji_plugin/rpmautospec_plugin.py %{buildroot}%{_libdir}/koji-hub-plugins/
 
-#%package -n rpmautospec-koji-plugin
-#Summary: Provides the koji plugin for autorel and autochangelog
-#Requires: python3-%{srcname} == %{version}-%{release}
-#Requires: koji
+%package -n python3-%{srcname}-koji-plugin
+Summary: Provides the koji plugin for autorel and autochangelog
+Requires: python3-%{srcname} == %{version}-%{release}
+Requires: koji
```

Comment:
Do we want to require koji or python/python3-koji?

Reply:
AIUI, the plugin needs python3-koji. We could require koji on top, just because the plugin is of little use without Koji (the builder), but this feels like more of a philosophical question. :wink: